dev/VSIX/Directory.Build.props

```diff
@@ -7,7 +7,7 @@
             https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json
         </RestoreSources>
         <CppWinRTVersion Condition="'$(CppWinRTVersion)' == ''">2.0.210806.1</CppWinRTVersion>
-        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.20348.19</WindowsSDKBuildToolsVersion>
+        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.22000.194/WindowsSDKBuildToolsVersion>
```

Comment:
typo:
```suggestion
        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.22000.194>/WindowsSDKBuildToolsVersion>
```

Reply:
Fix: https://github.com/microsoft/WindowsAppSDK/pull/1537

Reply:
Merged

Reply:
Good catch @asklar, and thanks @rohanp-msft!